lib/modules/datasource/nuget/index.ts

```diff
@@ -18,6 +18,17 @@ export class NugetDatasource extends Datasource {
 
   override readonly registryStrategy = 'merge';
 
+  override readonly releaseTimestampSupport = true;
+  override readonly releaseTimestampNote = `
+  For the v2 api, the release timestamp is determined from the \`Publised\` tag and,
```

Comment:
```suggestion
  For the v2 API, the release timestamp is determined from the \`Published\` tag and,
```

I think this is a typo? Also capitalize abbreviations like API.